receiver/googlecloudspannerreceiver/internal/metadataconfig/metadata.yaml

```diff
@@ -519,3 +519,22 @@ metadata:
         data:
           type: "gauge"
         unit: "second"
+#
+# -------------------------------------------- Table Size Stats ------------------------------------------------------
+#
+  - name: "hourly table sizes stats"
```

Comment:
```suggestion
  - name: "hourly table size stats"
```

Reply:
Yeah, I was literally thinking of that and had it plural because that's the name of the `SPANNER_SYS` table: `TABLE_SIZES_STATS_1HOUR`

Reply:
Yeah, it's a bit of an unclear case, I had to think about it for a minute. To me it makes more sense to massage the table name into something more grammatically correct since it's the display name.